holidayextras/ui-toolkit
# UXUI-204 grunt-release

#### What does this PR do?
Adds build tasks that:
1. Bump version (semver) in git tag & package.json
2. Commit & push all files & tag
3. Build & publish docs to `gh-pages` branch

#### What unit or integration tests does this PR have?
N/A

#### What selenium tests does this PR have?
N/A

#### How should a developer review this?
Sanity check the code & run the manual steps below

#### How should this be manually tested?
Release a patch from this branch:
```
npm install
grunt release-patch
```
You should see the patch version bumped & the docs dir pushed to `gh-pages`.

I will tidy up the tags after this PR is merged :)

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
[UXUI-204](https://holidayextras.jira.com/browse/UXUI-204)

#### Screenshots (if appropriate)
![console](https://cldup.com/p3Wjasstio.png)

#### What gif best describes how you feel about this work?
![lol](http://tclhost.com/BONR5Q6.gif)

---
#### Quality checklist (for PR author to complete BEFORE code review):

#### IP or Developer Review:

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
